src/public/stylesheets/terminal.grid.css

```diff
@@ -34,3 +34,74 @@
   grid-column: 1/3;
   grid-row: 2;
 }
+
+#TorqueGUI {
+  padding: 25px;
+  grid-column: 2 / 3;
+  grid-row: 1;
+  height: 100%;
+  display: grid;
+  gap: 30px;
+  grid-template-columns: 300px 300px 100px 100px;
+  grid-template-rows: 10px 50px 50px 50px;
+}
+#torqueInput {
```

Comment:
anything.grid.css is generally reseved for all CSSGrid templates. You should move your button styling and such to terminal.css